libeos-updater-flatpak-installer/tests/installer.c

```diff
@@ -519,11 +516,20 @@ main (int   argc,
 {
   setlocale (LC_ALL, "");
 
-  g_test_init (&argc, &argv, NULL);
+  g_test_init (&argc, &argv, G_TEST_OPTION_ISOLATE_DIRS, NULL);
 
   /* Since we setup a flatpak repo with the architecture being overridden
    * as "arch", we need to override it here too */
   g_setenv ("EOS_UPDATER_TEST_OVERRIDE_ARCHITECTURE", "arch", FALSE);
+  g_setenv ("FLATPAK_SYSTEM_HELPER_ON_SESSION", "1", TRUE);
+  g_setenv ("GIO_USE_VFS", "local", TRUE);
+
+  /* FIXME: The tests should theoretically be run with a fake D-Bus system bus
+   * and a load of mock services. However, those mock services have not been
+   * written yet (see https://phabricator.endlessm.com/T25340). In the meantime,
+   * to avoid behaviour differences in the tests when running on OBS/Jenkins vs
+   * a local VM, disable the system bus for everyone: */
+  g_setenv ("DBUS_SYSTEM_BUS_ADDRESS", "unix:/dev/null", TRUE);
```

Comment:
The spec seems to imply it should be `unix:path=/dev/null`: https://dbus.freedesktop.org/doc/dbus-specification.html#addresses

Reply:
Oops, that’s what happens when I think I can remember things rather than looking them up. Pushing a fix shortly, thanks.